src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -39,9 +39,8 @@
 import itertools
 import math
 import pivy.coin as coin
-import PySide.QtCore as QtCore
-import PySide.QtGui as QtGui
-
+import PySide2.QtCore as QtCore
+import PySide2.QtGui as QtGui
```

Comment:
We leave an empty line between system imports and our specific system imports.

Reply:
Implemented using `flake8-import-order`

Reply:
By the way, in FreeCAD we don't import `PySide2` directly because we have a fake `PySide` module that imports `PySide2`. This helps us keep compatibility with Python 2/Qt4, and Python 3/Qt5.

Once we completely ditch Python 2 and Qt4, then we could change `PySide` to `PySide2` everywhere, however, I don't think there is much benefit in doing this. Eventually we may have to support `PySide6` together with `PySide2` so the fake module could be retained.

Reply:
Reverted to PySide